t/unit/app/test_app.py

```diff
@@ -1399,6 +1401,190 @@ def test_send_task_expire_as_string(self):
         except TypeError as e:
             pytest.fail(f'raise unexcepted error {e}')
 
+    def test_native_delayed_delivery_countdown(self):
```

Comment:
I wonder if you should maybe move the delayed delivery unit tests to their own module, like the smoke tests also have their own test module for this grand feature.

If you’re already fixing the above comments, I highly recommend moving this to a dedicated test file to keep the test environment/codebase clean and easy to maintain.

I hate our super long test files, it’s very hard to locate stuff, so let’s not complicate it more.

Reply:
I see now that you did add such a file - why didn’t you include these tests there then?

Cosmetic issue though.

Reply:
Make sense, thanks for explaining!